Competition2018/src/main/java/competition/subsystems/gripperdeploy/commands/DeployDownCommand.java

```diff
@@ -0,0 +1,36 @@
+package competition.subsystems.gripperdeploy.commands;
+
+import com.google.inject.Inject;
+
+import competition.subsystems.gripperdeploy.GripperDeploySubsystem;
+import xbot.common.command.BaseCommand;
+
+public class DeployDownCommand extends BaseCommand {
```

Comment:
Let's name this `GripperDeployDownCommand` since there's more than one system on this robot that can "deploy"